frontend/src/Pages/AllUser/HomePage/HeroSection/Hero.jsx

```
@@ -0,0 +1,27 @@
+
+
+const Hero = () => {
+    return (
+        <div>
+            <div
+                className="hero h-[70vh]"
+                style={{
+                    backgroundImage: "url(https://i.ibb.co.com/JRs9429M/Whats-App-Image-2025-02-10-at-11-00-20-080a95b3.jpg)",
+                }}>
+                <div className="hero-overlay bg-opacity-60"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)